src/OpenyActivityFinderSolrBackend.php

```diff
@@ -160,9 +160,7 @@ public function doSearchRequest($parameters) {
     $query->getParseMode()->setConjunction('OR');
     $query->setParseMode($parse_mode);
```

Comment:
We came to the same changes yesterday - and we also removed all these $parse_mode manipulations (3 lines above). This direct parse mode is not really the thing we want to use here - the default 'terms' parse mode is working better for our case.

Reply:
Could you add a suggestion? @duozersk